eng/targets/ResolveReferences.targets

```
@@ -34,6 +34,7 @@
   <ItemGroup>
     <_ImplicitPackageReference Include="@(PackageReference->WithMetadataValue('IsImplicitlyDefined', 'true'))" />
     <_ExplicitPackageReference Include="@(PackageReference)" Exclude="@(_ImplicitPackageReference)" />
+    <OriginalReferences Include="@(Reference)" />
```

```suggestion
    <_OriginalReferences Include="@(Reference)" />
```

Suggest using underscores to indicate this isn't a thing people should put in the .csproj file.